### PR TITLE
sync with english version for `console.log`

### PR DESCRIPTION
**Pull request by Jungzl**
### Description

Align with original content.

### Motivation

zh-CN version has no `Logging objects` part.

### CODE REVIEW

File: files/zh-cn/web/api/console/log/index.md [hunk 1]
```diff
@@ -29,16 +29,25 @@ console.log(`temp 的值为：${temp}`)
...
 ## 规范
 
-不属于任何公开的规范
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
 
 ## 相关链接
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/api/console/log/index.md:50]:
```suggestion
## 参见
```

参考：https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md

File: files/zh-cn/web/api/console/log/index.md [hunk 2]
```diff
@@ -29,16 +29,25 @@ console.log(`temp 的值为：${temp}`)
 
 查看[向控制台输出文本](/zh-CN/docs/Web/API/Console#Outputting_text_to_the_console)来了解更多{{ domxref("console") }}对象的用法。
 
-## 浏览器兼容性
+## 返回值
 
-{{Compat}}
+None ({{jsxref("undefined")}})。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/api/console/log/index.md:34]:
```suggestion
无（{{jsxref("undefined")}}）。
```

File: files/zh-cn/web/api/console/log/index.md [hunk 3]
```diff
@@ -29,16 +29,25 @@ console.log(`temp 的值为：${temp}`)
...
+
+## 输出对象
+
+不要使用`console.log(obj)`，而是使用`console.log(JSON.parse(JSON.stringify(obj)))`。
+
+这样可以确保你所看到的`obj`的值是当前输出的值。否则，许多浏览器会提供一个随着值的变化而不断更新的实时视图。这可能不是你想要的。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/api/console/log/index.md:40]:
```suggestion
不要使用 `console.log(obj)`，而应该使用 `console.log(JSON.parse(JSON.stringify(obj)))`。

这样可以确保你所看到的 `obj` 的值是当前输出的值。否则，大多数浏览器会提供一个随着值的变化而不断更新的实时视图。这可能不是你想要的。
```

参考标点部分：https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#标点符号

**Review by yin1999 — APPROVED** [review]
Thanks @Jungzl @Jack-Zhang-1314.

**Pull request merged**